src/pyfmi/fmi.pyx

```diff
@@ -469,15 +482,19 @@ cdef class ModelBase:
         return self._default_options('pyfmi.fmi_algorithm_drivers', algorithm)
 
     def get_log_filename(self):
-        return decode(self._fmu_log_name)
+        """ Returns a name of the logfile, if logging is done to a stream it returns
+            a string formatted base on the model identifier.
+        """
+        return '{}_log'.format(self._modelid) if self.log_is_stream else decode(self._fmu_log_name)
```

Comment:
Why do we return a filename for when it is a stream? There is no file... (or at least any file that we know / or care about)

Reply:
I initially changed this function to return an empty string, but then when I looked at extract_xml_log, it says that the file it creates is based on what this function is returning and therefore I changed it to above.
I could however change this function to return an empty string instead and update to docstring in extract_xml_log, what approach do you think is the best?